docs/docs-beta/docs/dagster-plus/deployment/environment-variables/built-in.md

```diff
@@ -5,4 +5,33 @@ sidebar_position: 3
 sidebar_label: "Built-in variables"
 ---
 
-# Built-in environment variables
\ No newline at end of file
+
+[Dagster+](/todo) provides a set of built-in, automatically populated environment variables, such as the name of a deployment or details about a branch deployment commit, that can be used to modify behavior based on environment.
+
+### All deployment variables
+
+The following variables are available in every deployment of your Dagster+ instance, including full (e.g., `prod`) and branch deployments.
+
+| Key | Value |
+|---|---|
+| `DAGSTER_CLOUD_DEPLOYMENT_NAME` | The name of the Dagster+ deployment. <br/><br/>  **Example:** `prod`. |
+| `DAGSTER_CLOUD_IS_BRANCH_DEPLOYMENT` | `1` if the deployment is a branch deployment. |
+
+
+### Branch Deployment variables
+
+The following environment variables are available only in a [branch deployment](/todo).
+
+For every commit made to a branch, the following environment variables are available:
+
+| Key | Value |
+|---|---|
+| DAGSTER_CLOUD_GIT_SHA | The SHA of the commit. <br/><br/> **Example:** `5c5fa4643968a1b8043b58c159fb0600af8a35b2`. |
```

Comment:
I'm playing around with the idea of having examples for each of the env vars. I think for some of them it's kind of redundant (like email, pr message, etc). For ones like the timestamp though, I think it is really useful as a reader to know what format you're getting the data in (YYYY-MM-DD vs MM-DD-YYYY, vs a timestamp int, etc)

Does it feel odd to have examples for some but not all? I think the timestamp var is the only one that really needs it...

Reply:
Yea I tend to agree. I don't think they all need it. Although for Unix timestamp we should mention if its in seconds or ms, since its not always standardized.